### ISSUE TITLE
Keyboard Shortcuts: voiceover not reading the keyboard shortcuts

### ISSUE THREAD

**Issue by samwhale** [GoogleForCreators/web-stories-wp#8551]
## Bug Description

Keyboard shortcut list is not being read when using voiceover. Wondering if other assistive technologies will have the same problem.

## Expected Behaviour

Voiceover should read the keyboard shortcut list and it should be navigatable when using assistive technologies.

## Steps to Reproduce

1. If testing on mac, use`Cmd` + `F5` to open voice over.
2. Open the keyboard shortcuts menu
3. Try to move around the menu. Should see that you are stuck on the close button.

## Screenshots

![shortcut](https://user-images.githubusercontent.com/22185279/127919898-f718705c-f9b0-4dcc-9a1c-bc3f39ec7c4b.gif)

## Additional Context

- Plugin Version: 1.10.0-alpha.0+8ac805e
- WordPress Version: 5.8
- Operating System: MacOSX Big Sur 11.5.5
- Browser: Chrome

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by BrittanyIRL** [comment]
This feels tied to this: https://github.com/google/web-stories-wp/issues/7912 

Which I think we should try and fix soon for consistency with the popups.

**Comment by samwhale** [comment]
@BrittanyIRL good catch - this is a duplicate. I can close this in favor of the other issue
